rmw_cyclonedds_cpp/src/TypeSupport_impl.hpp

```diff
@@ -75,6 +75,34 @@ align_int_(size_t __align, T __int) noexcept
   return (__int - 1u + __align) & ~(__align - 1);
 }
 
+template<typename MembersType>
+void resize_field(
+  const MembersType * member,
+  void * field,
+  size_t size)
+{
+  if (!member->resize_function) {
+    throw std::runtime_error("unexpected error: resize function is null");
+  }
+
+  member->resize_function(field, size);
```

Comment:
so this code is only used for cpp introspection typesupport, right?

Reply:
Correct. In retrospect, I could use overloads instead of template specialization.

Reply:
> Correct. In retrospect, I could use overloads instead of template specialization.

:+1: that sounds better

Reply:
See https://github.com/ros2/rmw_cyclonedds/pull/313/commits/6a5b158025a264b1e397765d185c393ce310614c.